add-ons/APPLICATION_TEMPLATE.md

```diff
@@ -0,0 +1,182 @@
+# New Project Project Application
+
+Please see the [proposals/new-project-addition-process.md](Documentation) to
+better understand the full process for submitting a new project.
+
+In short, copy this Application Template and populate the document.
+--!>
+
+
+## Background information
+
+1. Submitter Name
+-
+
+2. Submitter’s relationship to project / title
+-
+
+3. Project Name
+-
+
+4. Why this project is valuable to the Kubeflow Community
+-
+
+5. Short Description / Functionality
+-
+
+6. Website
+-
+
+7. GitHub repository
+-
+
+8. 1st Release date
+-
+
+9. Project Meeting Times
+-
+
+10. Meeting Notes (link)
+-
+
+11. Installation Documentation (provide status and link)
+-
+
+12. Application Documentation (provide status and link)
+-
+
+13. Security Profile (CVE scanning)
+-
+
+14. Ownership / Legal Profile (license type, any potential issues for CNCF)
+-
+
+15. Kubernetes native (K8s version)
+-
+
+16. AuthN/Z, Isolation mechanisms
+-
+
+17. Project Roadmap
+-
+
+18. Other Information
+-
+
+## Metrics
+
+Number of Contributors
+- [ ] 1
+- [ ] <10
+- [ ] 10+
+
+Number of Users
+- [ ] 1
+- [ ] <10
+- [ ] 10+
+
+GitHub Pull Requests in the last 90 days
+- [ ] 1
+- [ ] <10
+- [ ] 10+
+
+Number of Releases
+- [ ] No release
+- [ ] 1 release
+- [ ] 2+ releases
+
+Downloads
+- [ ] <100
+- [ ] <10k
+- [ ] 10k+
+
+## Kubeflow Specific Questions
+
+1. Kubeflow specific Roadmap
+    - [ ] Yes
+    - [ ] No
+    - Link
+
+2. Workflows that integrate with Kubeflow Components
+- [ ] 1
+- [ ] 2
+- [ ] 3+
+
+3. Manifest Integration
+    - [ ] Yes
+    - [ ] No
+    - [ ] Planned
+
+4. Installation
+    - [ ] Testing
+    - [ ] Stand alone
+    - [ ] Part of Manifests
+    - [ ] Part of Distributions
+
+5. Application
+    - [ ] Testing
+    - [ ] Stand alone
+    - [ ] Part of Kubeflow workflow
+
+6. Installation Documentation (Current Quality)
+    - [ ] Good
+    - [ ] Fair
+    - [ ] Part of Kubeflow
+
+7. Application Documentation (Current Quality)
+    - [ ] Good
+    - [ ] Fair
+    - [ ] Part of Kubeflow
+
+8. Kubeflow Website Documentation
+    - [ ] Yes
+    - [ ] No
+    - Pull Request #
+
+9. Distributions position on added testing, docs
+    - [ ] 0 Distribution
+    - [ ] 1 Distribution
+    - [ ] 2+ Distributions
+
+10. Blog/Social Media announcements
+    - [ ] Yes
+    - [ ] No
+    - Pull Request #
+
+## Level of Integration and Support Sought by Kubeflow
+
+1. What level?
+    - [ ] Add-on
+    - [ ] Core
+
+2. IP Ownerhsip
+    - [ ] Stand Alone
+    - [ ] Merge into Kubeflow/CNCF
+
+3. Kubeflow Working Group
+    - [ ] None
+    - [ ] New
+    - [ ] Part of Existing
+
+4. Posting Meetings on Kubeflow Calendar
+    - [ ] Yes
+    - [ ] No
+
+5. Use of Zoom Accounts
+    - [ ] Yes
+    - [ ] No
+
+6. Repository Ownership
+    - [ ] Yes
+    - [ ] No
+
+7. Pull request Approvers
+    - [ ] Yes
+    - [ ] No
+
+8. Posting on Kubeflow Social Media, blog, and website
+    - [ ] Yes
+    - [ ] No
+    - [ ] Collaborate
+
```

Comment:
One question should be on list of maintainers and affiliation to prove diversity